README.md

```diff
@@ -291,11 +293,13 @@ $ friends add activity Got lunch with Earnest H and Earnest S. in the park. Man,
 Activity added: "2017-05-01: Got lunch with Earnest Hemingway and Earnest Shackleton in the park. Man, I like Earnest Hemingway but really love Earnest Shackleton."
 ```
 
-And locations will be matched as well:
+And locations or their alias will be matched as well:
 
 ```bash
 $ friends add activity Went swimming near atlantis with George.
 Activity added: "2017-01-06: Went swimming near Atlantis with George Washington Carver."
+$ friends add activity Had lunch in NYC with George.
```

Comment:
I love this example. Could you change `NYC` to `nyc` so we can also show that it matches case-insensitively as we do with `atlantis`?